charts/scalardl/values.yaml

```diff
@@ -245,3 +245,29 @@ ledger:
     certChainSecret: ""
     # -- Name of the Secret containing the private key file used for TLS communication.
     privateKeySecret: ""
+    certManager:
+      # -- Use cert-manager to manage private key and certificate files.
+      enabled: false
+      # -- Configuration of a certificate for self-signed CA.
+      selfSignedCaRootCert:
+        duration: "87600h0m0s"
+        renewBefore: "360h0m0s"
+      # -- Duration of a certificate.
+      duration: "87600h0m0s"
+      # -- How long before expiry a certificate should be renewed.
+      renewBefore: "360h0m0s"
+      # -- Configuration of a private key.
+      privateKey:
+        algorithm: ECDSA
+        encoding: PKCS1
+        size: 256
+      # -- List of key usages.
+      usages:
+        - server auth
+        - key encipherment
+        - signing
+      # -- Subject Alternative Name (SAN) of a certificate.
+      dnsNames:
+        - localhost
```

Comment:
I set `localhost` as a default value of the `dnsNames`. This is because:

- The `grpc_health_probe` command accesses ScalarDL Ledger by specifying `localhost` like `-addr=localhost:50051`.
- The `grpc_health_probe` command verifies the certificate by using `Subject Alternative Name (SAN)` instead of `Common Name (CN)` of the certificate. In other words, at least one SAN configuration is necessary. (I will update the document in another PR.)